### ISSUE TITLE
[CI] LocalTimeOffsetTests.testKnownMovesBackToPreviousDay fails

### ISSUE THREAD

**Issue by valeriy42** [elastic/elasticsearch#67930]
**Build scan**:

https://gradle-enterprise.elastic.co/s/qt3vtnsaiux6c

**Repro line**:

```
 ./gradlew ':server:test' --tests "org.elasticsearch.common.LocalTimeOffsetTests.testKnownMovesBackToPreviousDay" -Dtests.seed=A94578DD830A6882 -Dtests.security.manager=true -Dtests.locale=es-CU -Dtests.timezone=ART -Druntime.java=8 -Dtests.fips.enabled=true
```

**Reproduces locally?**:

No

**Applicable branches**:
`7.10`

**Failure history**:

https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P7D&search.timeZoneId=Europe/Berlin&tests.container=org.elasticsearch.common.LocalTimeOffsetTests&tests.sortField=FAILED&tests.test=testKnownMovesBackToPreviousDay&tests.unstableOnly=true

Failed 48 times in the past 4 days.

**Failure excerpt**:
```
org.elasticsearch.common.LocalTimeOffsetTests > testKnownMovesBackToPreviousDay FAILED
    java.lang.AssertionError: Europe/Paris just after 1911-03-1T00:01:00 should move back
        at __randomizedtesting.SeedInfo.seed([A94578DD830A6882:8389B2B7B97FA87F]:0)
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.assertTrue(Assert.java:41)
        at org.elasticsearch.common.LocalTimeOffsetTests.assertKnownMovesBacktoPreviousDay(LocalTimeOffsetTests.java:266)
        at org.elasticsearch.common.LocalTimeOffsetTests.testKnownMovesBackToPreviousDay(LocalTimeOffsetTests.java:256)
REPRODUCE WITH: ./gradlew ':server:test' --tests "org.elasticsearch.common.LocalTimeOffsetTests.testKnownMovesBackToPreviousDay" -Dtests.seed=A94578DD830A6882 -Dtests.security.manager=true -Dtests.locale=es-CU -Dtests.timezone=ART -Druntime.java=8 -Dtests.fips.enabled=true

Suite: Test class org.elasticsearch.common.LocalTimeOffsetTests
  1> [2021-01-25T18:14:21,354][INFO ][o.e.c.LocalTimeOffsetTests] [testGap] before test
  1> [2021-01-25T18:14:21,362][INFO ][o.e.c.LocalTimeOffsetTests] [testGap] after test
  1> [2021-01-25T18:14:21,369][INFO ][o.e.c.LocalTimeOffsetTests] [testNotFixed] before test
  1> [2021-01-25T18:14:21,371][INFO ][o.e.c.LocalTimeOffsetTests] [testNotFixed] after test
  1> [2021-01-25T18:14:21,380][INFO ][o.e.c.LocalTimeOffsetTests] [testUtc] before test
  1> [2021-01-25T18:14:21,383][INFO ][o.e.c.LocalTimeOffsetTests] [testUtc] after test
  1> [2021-01-25T18:14:21,389][INFO ][o.e.c.LocalTimeOffsetTests] [testAfterRules] before test
  1> [2021-01-25T18:14:21,399][INFO ][o.e.c.LocalTimeOffsetTests] [testAfterRules] after test
  1> [2021-01-25T18:14:21,412][INFO ][o.e.c.LocalTimeOffsetTests] [testJustTransitions] before test
  1> [2021-01-25T18:14:21,414][INFO ][o.e.c.LocalTimeOffsetTests] [testJustTransitions] after test
  1> [2021-01-25T18:14:21,417][INFO ][o.e.c.LocalTimeOffsetTests] [testLastTransitionOverlapsRules] before test
  1> [2021-01-25T18:14:21,419][INFO ][o.e.c.LocalTimeOffsetTests] [testLastTransitionOverlapsRules] after test
  1> [2021-01-25T18:14:21,422][INFO ][o.e.c.LocalTimeOffsetTests] [testRangeTooLarge] before test
  1> [2021-01-25T18:14:21,423][INFO ][o.e.c.LocalTimeOffsetTests] [testRangeTooLarge] after test
  1> [2021-01-25T18:14:21,427][INFO ][o.e.c.LocalTimeOffsetTests] [testKnownMovesBackToPreviousDay] before test
  1> [2021-01-25T18:14:21,431][INFO ][o.e.c.LocalTimeOffsetTests] [testKnownMovesBackToPreviousDay] after test
  2> REPRODUCE WITH: ./gradlew ':server:test' --tests "org.elasticsearch.common.LocalTimeOffsetTests.testKnownMovesBackToPreviousDay" -Dtests.seed=A94578DD830A6882 -Dtests.security.manager=true -Dtests.locale=es-CU -Dtests.timezone=ART -Druntime.java=8 -Dtests.fips.enabled=true
  2> java.lang.AssertionError: Europe/Paris just after 1911-03-1T00:01:00 should move back
        at __randomizedtesting.SeedInfo.seed([A94578DD830A6882:8389B2B7B97FA87F]:0)
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.assertTrue(Assert.java:41)
        at org.elasticsearch.common.LocalTimeOffsetTests.assertKnownMovesBacktoPreviousDay(LocalTimeOffsetTests.java:266)
        at org.elasticsearch.common.LocalTimeOffsetTests.testKnownMovesBackToPreviousDay(LocalTimeOffsetTests.java:256)
  1> [2021-01-25T18:14:21,454][INFO ][o.e.c.LocalTimeOffsetTests] [testOverlap] before test
  1> [2021-01-25T18:14:21,457][INFO ][o.e.c.LocalTimeOffsetTests] [testOverlap] after test
  1> [2021-01-25T18:14:21,472][INFO ][o.e.c.LocalTimeOffsetTests] [testSingleTimeBeforeRules] before test
  1> [2021-01-25T18:14:21,474][INFO ][o.e.c.LocalTimeOffsetTests] [testSingleTimeBeforeRules] after test
  1> [2021-01-25T18:14:21,493][INFO ][o.e.c.LocalTimeOffsetTests] [testSingleTimeAfterRules] before test
  1> [2021-01-25T18:14:21,503][INFO ][o.e.c.LocalTimeOffsetTests] [testSingleTimeAfterRules] after test
  1> [2021-01-25T18:14:21,541][INFO ][o.e.c.LocalTimeOffsetTests] [testTransitionsWithTransitionsAndRules] before test
  1> [2021-01-25T18:14:21,582][INFO ][o.e.c.LocalTimeOffsetTests] [testTransitionsWithTransitionsAndRules] after test
  1> [2021-01-25T18:14:21,585][INFO ][o.e.c.LocalTimeOffsetTests] [testLastTransitionWithoutRules] before test
  1> [2021-01-25T18:14:21,587][INFO ][o.e.c.LocalTimeOffsetTests] [testLastTransitionWithoutRules] after test
  1> [2021-01-25T18:14:21,590][INFO ][o.e.c.LocalTimeOffsetTests] [testFixedOffset] before test
  1> [2021-01-25T18:14:21,591][INFO ][o.e.c.LocalTimeOffsetTests] [testFixedOffset] after test
  1> [2021-01-25T18:14:21,614][INFO ][o.e.c.LocalTimeOffsetTests] [testJustOneRuleApplies] before test
  1> [2021-01-25T18:14:21,616][INFO ][o.e.c.LocalTimeOffsetTests] [testJustOneRuleApplies] after test
  2> NOTE: leaving temporary files on disk at: /dev/shm/elastic+elasticsearch+7.10+matrix-java-periodic-fips/ES_RUNTIME_JAVA/zulu8/nodes/general-purpose/server/build/testrun/test/temp/org.elasticsearch.common.LocalTimeOffsetTests_A94578DD830A6882-001
  2> NOTE: test params are: codec=Asserting(Lucene87): {}, docValues:{}, maxPointsInLeafNode=418, maxMBSortInHeap=6.708476269400118, sim=Asserting(RandomSimilarity(queryNorm=true): {}), locale=es-CU, timezone=ART
  2> NOTE: Linux 4.19.0-13-cloud-amd64 amd64/Azul Systems, Inc. 1.8.0_282 (64-bit)/cpus=32,threads=1,free=260117056,total=522190848
  2> NOTE: All tests run in this JVM: [TransportServiceHandshakeTests, RemoteClusterClientTests, ThreadPoolSerializationTests, SnapshotsInProgressSerializationTests, SuggestionTests, CompletionSuggesterBuilderTests, SliceBuilderTests, QueryPhaseTests, GeoShapeQueryTests, ScriptValuesTests, MovAvgTests, BucketScriptTests, SumTests, InternalValueCountTests, InternalCardinalityTests, AvgAggregatorTests, RareTermsAggregatorTests, BestDocsDeferringCollectorTests, InternalReverseNestedTests, DoubleBoundsTests, GeoHashGridTests, CompositeAggregatorTests, ScriptServiceTests, RestSearchActionTests, RestBulkActionTests, RestCreateIndexActionTests, BlobStoreRepositoryRestoreTests, EnableAssignmentDeciderTests, FsHealthServiceTests, SystemIndexDescriptorTests, IndicesRequestCacheTests, TermVectorsServiceTests, ShardPathTests, RetentionLeaseXContentTests, SearchStatsTests, ReindexRequestTests, SpanTermQueryBuilderTests, RangeQueryBuilderTests, MatchAllQueryBuilderTests, FunctionScoreEquivalenceTests, MergeSchedulerSettingsTests, RangeFieldQueryStringQueryBuilderTests, LegacyTypeFieldMapperTests, IdFieldMapperTests, DoubleIndexingDocTests, AllFieldMapperTests, SingleOrdinalsTests, ReadOnlyEngineTests, PreConfiguredTokenFilterTests, PrimaryShardAllocatorTests, NodeEnvironmentTests, FileBasedSeedHostsProviderTests, FilterPathGeneratorFilteringTests, LongObjectHashMapTests, KeyedLockTests, ArrayUtilsTests, SettingsModuleTests, LockedRecyclerTests, ShardCoreKeyMapTests, LocalTimeOffsetTests]

Tests with failures:
 - org.elasticsearch.common.LocalTimeOffsetTests.testKnownMovesBackToPreviousDay

```

**Comment by nik9000** [comment]
I believe I dropped this test in newer branches because not all machines have that sweet sweet 1911 DST data. I had wanted a test that verified that my code works with it, and, well, it does, but not all machines know anything about 1911. Oh well. I'll drop the test from that branch.

**Comment by nik9000** [comment]
Closed by backporting 9ba0c53bc89bba60a016bc0db3347667c4c56037.
